### PR TITLE
Map 9 pois

### PR DESCRIPTION
**Pull request by j-adamcic**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
```
Comment (PaulVII) [inline db/seeds.rb:7]: why did you add this? It doesn't seem to be relevant at all to the program

File: db/seeds.rb [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+# Points of Interest
+PointOfInterest.create(:point => Point.create(:x => 13.131646, :y => 52.393869), :description => 'This is Mr. Net, a landmark of the HPI.', :name => 'Mr. Net')
+PointOfInterest.create(:point => Point.create(:x => 13.132215, :y => 52.393793), :description => 'This is Lake HPI.', :name => 'Lake HPI')
+PointOfInterest.create(:point => Point.create(:x => 13.133757, :y => 52.394414), :description => 'This area is used for freetime activities by HPI students.', :name => 'Meadow')
+PointOfInterest.create(:point => Point.create(:x => 13.13130, :y => 52.39335), :description => 'This is a nice place to eat.', :name => 'Ulf''s Cafe')
```
Comment (PaulVII) [inline db/seeds.rb:13]: these lines create a ton of linter issues. I'd recommend to fix these if they're not false positives or otherwise undesirable

File: app/views/map/index.html.erb [hunk 3]
```diff
@@ -8,6 +8,8 @@
 	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/leaflet.locatecontrol@0.76.0/dist/L.Control.Locate.min.css" />
   	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.3.1/leaflet.css" />
 	
+	
+	<%= content_tag :div, '', id: 'container', data: { source: @PoI } %>
```
Comment (PaulVII) [inline app/views/map/index.html.erb:12]: 'Cotainer' is basically just what a div is, I think that name is a little too unspecific here, especially since IDs need to be unique and the term is widely used in web apps. A Maybe 'poi-data' or something similar? (just you don't forget: this would need to be changed in map_build l. 85  as well)

**Review by PaulVII — APPROVED** [review]
I think it is good to go, but I'd appreciate if you considered my minor (non-blocking) comments. It'd be great if this code didn't add new linting errors which cause a failed pipeline.

File: app/assets/javascripts/OutdoorMap/map_build.js [hunk 4]
```diff
@@ -80,20 +80,32 @@ for (const feature of buildings) {
     layers[feature.properties.campus].addLayer(layer);
 }
 
+//Add points of interest
+layers['Points of Interest'] = L.layerGroup().addTo(mymap);
+let pois = JSON.parse(document.getElementById('container').dataset.source);
+for(const feature of pois) {
```
Comment (23mafi) [inline app/assets/javascripts/OutdoorMap/map_build.js:86]: I would rename the variable feature to interestedPoint or similar, as feature would rather translate to Funktion, Eigenschaft oder Merkmal (for me the context does not quite fit)

**Review by 23mafi — COMMENTED** [review]
Code looks for me ok with Pauls advices. But unfortunately I can't see the pois on the map and have no idea why. Did I forget to run something else in addition to the migration?

**Comment by PaulVII** [comment]
> Code looks for me ok with Pauls advices. But unfortunately I can't see the pois on the map and have no idea why. Did I forget to run something else in addition to the migration?

Yeah, took me some time as well^^ you need to run `rails db:seed` to add the example POIs to the database

**Review by PaulVII — APPROVED** [review]

**Review by PaulVII — APPROVED** [review]
